2020-InfiniteRecharge/src/main/java/frc/robot/subsystems/PowerCell.java

```diff
@@ -0,0 +1,249 @@
+        shooter.config_kP(0, shooter_kP);
+        shooter.config_kD(0, shooter_kD);
+        shooter.config_kI(0, shooter_kI);
+        shooter.configNominalOutputForward(0, TIMEOUT);
+        shooter.configNominalOutputReverse(0, TIMEOUT);
+        shooter.configPeakOutputForward(+1, TIMEOUT);
+        shooter.configPeakOutputReverse(-1, TIMEOUT);
+        shooter.setNeutralMode(NeutralMode.Coast);
+    
+        //Feeder motor config
+        feederMotor.set(ControlMode.PercentOutput, 0);	
+        feederMotor.configClosedloopRamp(cLR, TIMEOUT);
+        feederMotor.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
+        feederMotor.config_kF(0, feeder_kF);
+        feederMotor.config_kP(0, feeder_kP);
+        feederMotor.config_kD(0, feeder_kD);
+        feederMotor.config_kI(0, feeder_kI);
+        feederMotor.configNominalOutputForward(0, TIMEOUT);
+        feederMotor.configNominalOutputReverse(0, TIMEOUT);
+        feederMotor.configPeakOutputForward(+1, TIMEOUT);
+        feederMotor.configPeakOutputReverse(-1, TIMEOUT);
+        feederMotor.setNeutralMode(NeutralMode.Brake);
+
+        //Storage Motor Config
+        storageMotorH.set(ControlMode.PercentOutput, 0);	
+        storageMotorH.configClosedloopRamp(cLR, TIMEOUT);
+        storageMotorH.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
+        storageMotorH.config_kF(0, storage_kF);
+        storageMotorH.config_kP(0, storage_kP);
+        storageMotorH.config_kD(0, storage_kD);
+        storageMotorH.config_kI(0, storage_kI);
+        storageMotorH.configNominalOutputForward(0, TIMEOUT);
+        storageMotorH.configNominalOutputReverse(0, TIMEOUT);
+        storageMotorH.configPeakOutputForward(+1, TIMEOUT);
+        storageMotorH.configPeakOutputReverse(-1, TIMEOUT);
+        storageMotorH.enableCurrentLimit(true);
+		storageMotorH.configPeakCurrentLimit(75,TIMEOUT);
+		storageMotorH.configContinuousCurrentLimit(40, TIMEOUT);
+		storageMotorH.configPeakCurrentDuration(1800,TIMEOUT);
+        storageMotorH.setNeutralMode(NeutralMode.Coast);
+
+        storageMotorL.set(ControlMode.PercentOutput, 0);	
+        storageMotorL.configClosedloopRamp(cLR, TIMEOUT);
+        storageMotorL.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
+        storageMotorL.config_kF(0, storage_kF);
+        storageMotorL.config_kP(0, storage_kP);
+        storageMotorL.config_kD(0, storage_kD);
+        storageMotorL.config_kI(0, storage_kI);
+        storageMotorL.configNominalOutputForward(0, TIMEOUT);
+        storageMotorL.configNominalOutputReverse(0, TIMEOUT);
+        storageMotorL.configPeakOutputForward(+1, TIMEOUT);
+        storageMotorL.configPeakOutputReverse(-1, TIMEOUT);
+        storageMotorL.enableCurrentLimit(true);
+		storageMotorL.configPeakCurrentLimit(75,TIMEOUT);
+		storageMotorL.configContinuousCurrentLimit(40, TIMEOUT);
+		storageMotorL.configPeakCurrentDuration(1800,TIMEOUT);
+        storageMotorL.setNeutralMode(NeutralMode.Coast);
```

Comment:
Someday (not in this PR) we should consider factoring this out into a separate class that known how to set up a Talon in velocity mode, with config settings in Shuffleboard.